src/test/java/teammates/test/cases/action/InstructorFeedbackResponseCommentAddActionTest.java

```diff
@@ -308,4 +322,19 @@ protected void testAccessControl() throws Exception {
         // remove the comment
         frcDb.deleteEntity(comment);
     }
+
+    /**
+     * Filters instructor comment according to comment text from all comments on a response.
+     *
+     * @param responseId response id of response
+     * @param commentText comment text
+     * @return feedback participant comment
+     */
+    private FeedbackResponseCommentAttributes getInstructorComment(String responseId, String commentText) {
+        FeedbackResponseCommentsDb frcDb = new FeedbackResponseCommentsDb();
+        List<FeedbackResponseCommentAttributes> frcList = frcDb.getFeedbackResponseCommentsForResponse(responseId);
+        return frcList.stream()
+                       .filter(comment -> comment.commentText.getValue().equals(commentText))
+                       .collect(Collectors.toList()).get(0);
```

Comment:
Should this method return a list instead of getting the first comment?

Reply:
Why? I think since it is only used in test, and to get instructor comment for verifying, how would it make difference to return list or comment?

Reply:
Not friendly for future development. If you name the method as `getInstructorComments`, then you shall return a list. If you insist your implementation, you shall name the method as `getFirstInstructorComment`. In addition, someone why try to use your method will get NPE if there is no comment.

Reply:
Okay,  I'll change it.